### DISCUSSION TITLE
install Pythonnet without internet issue

### DISCUSSION THREAD

**Discussion by QUESADADEVELOPER** [pythonnet/pythonnet#2121] [General]
Build and Install Pythonnet From Source:

Download the tar.gz source code archive from https://pypi.org/project/pythonnet/#files - done

Extract the archive - done

Comment out line 424: subprocess.check_call(cmd, shell=use_shell) -how do I do this? getting a lot of different answers? It looks like there is a file that I edit to comment it out, but I am not seeing the file.

thank you !!

**Comment by filmor** [comment]
Download the wheel file (and the one for clr_loader) and install that. To compile, you'll need internet.

**Reply by QUESADADEVELOPER** [reply]
That worked !!! thank you!!
